content/docs/1.6.0/deploy/important-notes/index.md

```diff
@@ -136,6 +137,31 @@ done
 
 Once you successfully upgrade to version v1.6.0, you will be able to view information about engine image versions on the UI.
 
+### Danger Zone Setting Configuration
+
+Starting with Longhorn v1.6.0, Longhorn allows you to promptly update the [Danger Zone settings](https://longhorn.io/docs/1.6.0/references/settings/#danger-zone) without the need to wait for all volumes detached. This enhancement is designed to provide users with more flexibility and efficiency in managing their configurations. To achieve the immediate effect, one of the following conditions must be met:
+
+- No Attached Volumes: If there are no volumes attached, the setting change will take effect immediately.
+- Engine Image Upgrade (Volume Live Upgrade): During a volume live upgrade, which involves creating a new instance manager pod, the setting change will take effect immediately for the new pod.
+
+If immediate effect conditions aren't met, the setting change is reconciled hourly, taking effect when all volumes are detached.
+
+- When all volumes are detached, the settings in the following table are immediately applied and the system-managed components are restarted. If you do not detach all volumes before the settings are synchronized, the settings are not applied immediately as mentioned above and you might need to reconfigure the settings after detaching the remaining volumes; otherwise, you can wait for the setting change to be reconciled in an hour.
```

Comment:
> If you do not detach all volumes before the settings are synchronized, the settings are not applied immediately, as mentioned above, and you might need to reconfigure the settings after detaching the remaining volumes. Otherwise, you can wait for the setting change to be reconciled in an hour.

Try to understand the case you mentioned

- Some v1 volumes are attached
- User updates the guaranteed cpu setting. The change cannot immediately take effect
- User detaches all v1 volumes
- What will happen? Which one is correct?
   - The change immediately take effect
   - User needs to wait for the reconciliation in one hour

Reply:
Usually second one, User needs to wait for the reconciliation in an hour.
It is for system-managed components except the Instance Manager.
The Instance Manager will be restarted when volumes are all detached.